lib/capabilities.js

```diff
@@ -41,7 +41,7 @@ var capabilities = {
       },
       {
         check: function(cb) {
-          if (agent().internal.supports.watchdog) {
+          if (agent().internal.supports.watchdog) { // XXX(sam) appmetrics?
```

Comment:
appmetrics won't run without its compiled addon, so some of the checks in here can be hard-coded to true, and some other capabilities (tracing) aren't supported by appmetrics now, so will be false. `agent().internal.supports.watchdog` will probably fail, there is no `.supports` in the adapter shim (but the conditional should just be rewritten)